uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/screens/habits/show/views/TargetCard.kt

```diff
@@ -51,37 +52,64 @@ class TargetCardPresenter {
                 isNumerical = habit.isNumerical
             ).firstOrNull()?.value ?: 0
 
+            val skippedDayToday = entries.countSkippedDays(
+                truncateField = DateUtils.TruncateField.DAY,
+                isNumerical = habit.isNumerical
+            ).firstOrNull()?.value ?: 0
+
```

Comment:
Not a required change, but this function is now looking very repetitive, especially after the changes introduced in this PR. Perhaps we could rewrite it.

Reply:
Maybe I could add a skippedDays: Int field to the Entry class, and then count these days in the List<Entry>.groupedSum?
My only issue with that is that this field would be only used when creating the TargetChart, so it wouldn't be great to have this almost never used field in the class 😄 
Another solution would be to create a class identical to Entry, but extended with the skippedDays field, create a .groupedSum method for it and use it in the TargetChart creation. It would be nice to just make a class that inherits from Entry, but I don't think it's best practice to use inheritance with data classes.
Should I try to implement any of these solutions?

Reply:
Let's just keep it like this for now.